internal/resolve/gosource/gosource_test.go

```diff
@@ -50,6 +60,14 @@ func TestResolve(t *testing.T) {
 			cwd, err := os.Getwd()
 			require.NoError(t, err)
 
+			if os.Getenv("GOCACHE") == "" {
+				os.Setenv("GOCACHE", t.TempDir())
+			}
+
+			if !containsGoPath(testCfg.Cfg.Environment) {
+				testCfg.Cfg.Environment = append(testCfg.Cfg.Environment, fmt.Sprintf("GOPATH=%s", os.Getenv("GOPATH")))
+			}
```

Comment:
Why is the `GOPATH` environment variable set if it's unset?
Shouldn't either be go modules be used, then GOPATH is ignored or go modules are not used and GOPATH should be set in the `test_config.json files`

Reply:
Without specifying a GOPATH value this error occurs `exec: running 'C:\Go\bin\go.exe env -json' in directory '.' exited with code 1, expected 0, output: 'missing $GOPATH'`
The reason it only sets a value if it's unset is because some tests already specify a GOPATH value.

Reply:
>  The reason it only sets a value if it's unset is because some tests already specify a GOPATH value.

But what is the reason that GOPATH must be set at all?
If the GOPATH environment variable is empty go determines a reasonable default on it's own normally.
Example on Linux:
```sh
$ unset GOPATH; go env GOPATH
/home/fho/go
```

Reply:
It failed because the `USERPROFILE` environment variable was not made available when running (`go env`).
(https://github.com/golang/go/wiki/SettingGOPATH).
I fixed it in https://github.com/simplesurance/baur/pull/241/commits/c6d2f22d683dd3c2871795c38a3270366db0303b and reverted changing the testcases